hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieROTablePathFilter.java

```diff
@@ -175,8 +181,12 @@ public boolean accept(Path path) {
             metaClientCache.put(baseDir.toString(), metaClient);
           }
 
-          fsView = FileSystemViewManager.createInMemoryFileSystemView(engineContext,
-              metaClient, HoodieInputFormatUtils.buildMetadataConfig(getConf()));
+          fsView = hoodieTableFileSystemViewCache.get(baseDir.toString());
```

Comment:
here the `fsView` would never get updated once put into cache, even the writer commits new file, the new file will not appears in the `fsView`, I think it may lead to wrong results in flink streaming read, right?

Reply:
Flink streaming reader does not use this code path.

Reply:
it is in flink incremental read code path. still a problem?

Reply:
Thanks for your review. 
As we can see HoodieROTablePathFilter already **create and cache** `HoodieTableMetaClient` at baseDir level, also `setLoadActiveTimelineOnLoad(true)` which will create an active timeline in singleton mode.
So that IMO no matter we cache the fsView or not, any new created files will not appear in current hoodieROTablePathFilter.

Now  we cached the fsView using above cached meta client and cached active timeline. Maybe can have no bad effect but can reduce unnecessary init action.

Reply:
Feel free to correct me if I am wrong :)